nan.h

```diff
@@ -1008,10 +1031,13 @@ class NanEscapableScope {
     delete[] data;
```

Comment:
Is this correct? I though smalloc used `malloc`, which means that this should be `free(data)`.